src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -38,12 +71,15 @@ import braintreeDepositAndCheckout from '@/graphql/mutation/braintreeDepositAndC
 import KvButton from '@/components/Kv/KvButton';
 import KvIcon from '@/components/Kv/KvIcon';
 import BraintreeDropInInterface from '@/components/Payment/BraintreeDropInInterface';
+import KvCheckbox from '@/components/Kv/KvCheckbox';
+import { email, getFailures } from '@/util/validators';
```

Comment:
Please use [vuelidate](https://vuelidate.js.org/) for validating the form inputs. You can see example usage in the [Monthly Good Setup page](https://github.com/kiva/ui/blob/master/src/pages/MonthlyGood/MonthlyGoodSetupPage.vue)

Reply:
👍